src/pages/task/taskTabs/buildBaronAndAnnotations/FileTicketButton.tsx

```diff
@@ -49,23 +49,23 @@ const FileTicketButton: React.VFC<FileTicketProps> = ({
   return (
     <Container>
       <Popconfirm
-        title="Do you want to create a failure ticket for this task?"
+        align="right"
         onConfirm={onClickFile}
-        icon={null}
-        placement="right"
-        okText="File Ticket"
-        okButtonProps={{ loading: loadingFileJiraTicket }}
-        cancelButtonProps={{ disabled: loadingFileJiraTicket }}
+        confirmText="File Ticket"
```

Comment:
nit. change casing maybe?

```suggestion
        confirmText="File ticket"
```